ganga/GangaCore/GPIDev/Base/Objects.py

```diff
@@ -687,6 +689,8 @@ def __init__(cls, name, bases, this_dict):
             this_schema.createDefaultConfig()
 
 
+@cpu_profiler
+@mem_profiler
```

Comment:
Is there a performance price to pay for adding these two, even if the cpu and memory profiler is turned off?

Reply:
There is no performance lag on my machine. However, we should test this in real world world scenarios just to be sure.

Reply:
No shouldn't really be as these are only run once at module import time and are essentially a no-op when deactivated